api/src/main/java/org/openmrs/module/fhir/api/util/FHIRObsUtil.java

```diff
@@ -262,18 +262,29 @@ public static Obs generateOpenMRSObs(Observation observation, List<String> error
 		Date instant = observation.getIssued();
 		obs.setDateCreated(instant);
 		
-		String conceptUuid = null;
+		String conceptCode = null;
+		String system = null;
+		Concept concept = null;
 		try {
 			CodeableConceptDt dt = observation.getCode();
 			List<CodingDt> dts = dt.getCoding();
 			CodingDt coding = dts.get(0);
-			conceptUuid = coding.getCode();
+			conceptCode = coding.getCode();
+			system = coding.getSystem();
 		}
```

Comment:
There could be multiple system values and code values right? Have to address that